plugins/scaffolder/src/components/JobStatusModal/JobStatusModal.tsx

```diff
@@ -46,17 +51,8 @@ export const JobStatusModal = ({ job, toCatalogLink }: Props) => {
     }
   };
 
-  const onClose = useCallback(() => {
-    if (!job) {
-      return;
-    }
-    if (job.status === 'COMPLETED' || job.status === 'FAILED') {
```

Comment:
This logic still needs to exist somewhere, as the onclose handler is called even when you accidentally click outside the modal. We don't want to close the modal accidentally.

Reply:
This function will only close the modal if the job is completed or failed.

Reply:
Maybe we can move this state to the outside, this component which is just a prop saying `closable` or something?

Reply:
Aha, so that's the catch! Knew something was up :P

Reply:
Makes sense to keep this here as of now. Will revert bits of commit.